src/main/java/racingcar/domain/RacingGame.java

```diff
@@ -0,0 +1,61 @@
+package racingcar.domain;
+
+import racingcar.exception.NoCarsExistException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+    private static final int DEFAULT_START_LINE = 0;
+    private static final int MOVABLE_BOUND = 4;
+
+    private final List<Car> cars;
+    private final NumberGenerator numberGenerator;
+    private final Coin gameCoin;
+
+    public RacingGame(List<String> splitCarNames, int gameTry, NumberGenerator numberGenerator) {
+        cars = splitCarNames.stream()
```

Comment:
다른 클래스의 생성자에서는 this를 붙였는데 여기서는 빠졌네요.
일관성 있는 코드 스타일을 유지하시기 바랍니다.

```suggestion
        this.cars = splitCarNames.stream()
```

Reply:
생성자 뿐만 아니라, 모든 인스턴스 필드에 `this` 키워드를 추가했습니다.

IDE를 사용하지 않는 경우(ex: 깃허브) 인스턴스 필드를 색깔 등으로 표현해주지 않아,
this 키워드를 통해 가독성을 높일 수 있다고 생각하여 사용하고 있는 방법입니다.

감사합니다!